src/main/java/de/tum/in/www1/artemis/web/rest/ResultResource.java

```diff
@@ -276,4 +278,20 @@ public ResponseEntity<Result> createResultForExternalSubmission(@PathVariable Lo
         return ResponseEntity.created(new URI("/api/results/" + savedResult.getId()))
                 .headers(HeaderUtil.createEntityCreationAlert(applicationName, true, ENTITY_NAME, savedResult.getId().toString())).body(savedResult);
     }
+
+    /**
+     * GET /exercises/:exerciseId/feedback-details : Retrieves all aggregated feedback details for a given exercise.
+     * The feedback details include counts and relative counts of feedback occurrences, along with associated test case names and task numbers.
+     *
+     * @param exerciseId The ID of the exercise for which feedback details should be retrieved.
+     * @return A ResponseEntity containing a list of FeedbackDetailDTOs, each with aggregated feedback details, including count, relative count,
+     *         detail text, test case name, and task number.
```

Comment:
I think it is better to link the respective type 

```suggestion
     * @return A ResponseEntity containing a list of {@link FeedbackDetailDTO}s
```